lib/validatex/validation.ex

```diff
@@ -107,10 +151,58 @@ defmodule Validatex.Validation do
     end
   end
 
+  @doc """
+  Verification if `field` has valid value.
+  """
   @spec valid?(field(any(), any())) :: boolean()
   def valid?({:field, _, {:valid, _}}), do: true
   def valid?(_), do: false
 
+  @doc """
+  Runs validation for `field` with `on_blur` event action.
+  """
+  def validate_on_blur(map, field, validator) when is_map(map) and key?(field) do
+    Map.update!(
+      map,
+      field,
+      &validate(&1, validator, :on_blur)
+    )
+  end
+
+  @doc """
+  Runs validation for `field` with `on_change` event action.
+  """
+  def validate_on_change(map, field, value, validator) when is_map(map) and key?(field) do
+    Map.update!(
+      map,
+      field,
+      &validate(&1, validator, {:on_change, value})
+    )
+  end
+
+  @doc """
+  Runs validation for `field` with `on_related_change` event action.
+  """
+  def validate_on_related_change(map, field, validator) when is_map(map) and key?(field) do
```

Comment:
Zde nevim jak ma byt pouzita tato fce v realnem pripade. viz https://github.com/iodevs/validatex_example

Tj. jak ji navazat na `handle_event` v `new_live.ex` tak abych docili stejne funkcionality jako mame u elm-validate.

Reply:
Ted zkousim pro formular napsat FSM (ted stavovy stroj), coz je asi nejlepsim resenim. Tedy resim to v `validatex_example` a az to budu mit doladene, tak bych to doresil tady, tak zatim to nemusis resit.

Reply:
Oki diky